students/TylerRay/session7/html_render.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
```

Comment:
There are a couple of Syntax errors here which I was able to quickly resolve and run. I will give partial credit for what you've completed thus far, the render isn't fully working for me.